starknet_py/net/models/transaction.py

```diff
@@ -105,6 +124,12 @@ def calculate_hash(self, chain_id: StarknetChainId) -> int:
             nonce=self.nonce,
         )
 
+    @staticmethod
+    def _contract_attribute_name(data: Dict[str, Any]):
+        return (
+            "contract_definition" if "contract_definition" in data else "contract_class"
+        )
```

Comment:
I wanted to test serializing contract that has `contract_definition` defined, but I went through a couple of old compiled contract I had and none of them had this. It must have been changed a while ago or I am looking in the wrong place.